chunk 1: Always format .Pa the same way, as requested by jmc@ (local change).

Index: tmac/doc.tmac
--- tmac/doc.tmac.orig
+++ tmac/doc.tmac
@@ -3761,8 +3761,6 @@
 .
 .        if \n[doc-in-files-section] \{\
 .          ds doc-saved-Pa-font "\*[doc-Pa-font]
-.          if n \
-.            ds doc-Pa-font "\*[doc-No-font]
 .        \}
 .
 .        ie (\n[doc-type1] == 1) \
